packages/fe-fpm-writer/test/test-input/integration/custom-page-app/js/webapp/test/locate-reuse-libs.js

```diff
@@ -0,0 +1,195 @@
+(function (sap) {
```

Comment:
We don't need that file in our test folder.

Reply:
Fixed that for you in https://github.com/SAP/open-ux-tools/pull/1056/commits/3c4194db211c0d2e26eb8ed659ccf1aa72de09bb